zap/src/main/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRF.java

```diff
@@ -196,6 +197,11 @@ protected AntiCsrfParam getParam() {
         return this.getParam().getTokensNames();
     }
 
+    /** Returns true if should detect CSRF variables by searching for partial matches. */
+    public Boolean getIsPartialMatchingEnabled() {
```

Comment:
Does it need to be public? The following could be called directly in the method.